tests/core/policies/test_ted_policy.py

```diff
@@ -74,14 +80,21 @@ def get_checkpoint_dir_path(train_path: Path, ted_pos: Optional[int] = 0) -> Pat
         train_path: the path passed to model_training.train_core for training output.
         ted_pos: the position of TED in the policies listed in the config.
     """
-    policy_dir_name = Path("policy_{}_{}".format(ted_pos, TEDPolicy.__name__))
+    policy_dir_name = Path("policy_{}_{}".format(ted_pos, Rasa2TEDPolicy.__name__))
     policy_path = train_path / DEFAULT_CORE_SUBDIRECTORY_NAME / policy_dir_name
     return policy_path / "checkpoints"
 
 
-def test_diagnostics():
+def test_diagnostics(
+    default_model_storage: ModelStorage, default_execution_context: ExecutionContext
+):
     domain = Domain.from_yaml(DOMAIN_YAML)
-    policy = TEDPolicy()
+    policy = TEDPolicy(
+        TEDPolicy.default_config.copy(),
+        default_model_storage,
+        Resource(""),
```

Comment:
```suggestion
        Resource("TEDPolicy"),
```

I know it doesn't make a huge difference, but just to make it slightly less likely something weird happens down the line.